docs/glossary.tsv

```diff
@@ -0,0 +1,46 @@
+Dataset Model	A dataset model is a formal representation devised for a dataset	HCA,JSON	Phillipe		
+Domain Entity	A domain entity is a information element (a type, a concept) core to the description of a given knowledge area	MAGE-OM	Phillipe		
+Domain model	The domain model is a  formal representation of a knowledge domain with concepts, roles, datatypes, individuals, and rules, typically grounded in a description logic.		Yojana	WikiPedia (https://en.wikipedia.org/wiki/Domain_model) 	
+FAIR sharing	FAIR sharing is a data use case that is intended for a data object (dataset) to exhibit a state of FAIRness purposed for sharing and re-use				
+Field level metadata	The filed level metadata descibed in detail about the data stored in the field either based on the type of data, the usage of data, the data source etc.		Yojana		
+General purpose metadata schema	Generalised schemas require limited time and expertise, but the description may be inadequate. They include what are considered the essential elements to describe any data. Such schemas usually not domain-specific.	JSON,TSV,CSV	Yojana	Research data management (https://libguides.anu.edu.au/c.php?g=881167&p=6358464)	
+Generic Data Model	Generic data models are generalizations of conventional data models that define standardised general relation types, together with the data that may be related or connected by such a relation type.		Yojana	WikiPedia (https://en.wikipedia.org/wiki/Generic_data_model)	
+Identifiable dataset	Identifiable dataset is any information (personal or indirect) that can link a dataset to its research study.		Yojana		
+Linked data representation	Linked Data is a set of design principles for sharing machine-readable interlinked data on the Web. When combined with Open Data (data that can be freely used and distributed), it is called linked data representation	Graph data representation, Database structures	Yojana	Ontotext (https://www.ontotext.com/knowledgehub/fundamentals/linked-data-linked-open-data/)	Linked Open Data (LOD)
+Machine-Interpretable Format	A format in which the computer/machine can interpret the underlying data similar to a human and dedude conclusions on the dataset based on human's questions		Yojana	Ontotext (https://www.ontotext.com/knowledgehub/fundamentals/what-is-the-semantic-web/)	
+Machine-Readable Format	A machine readable format is a information representation specifying how to serialize (write) information/data in ways software agents can handle	TIFF,CSV	Phillipe		computer-readable data
+Master Data Entity	The conceptual entity about which the observed or measured data is reported in the dataset(s). E.g., chemical compound, biospecimen, human subject, animal subject, cell-line 		Phillipe		
+Metadata Record	A metadata record is an instance of a metadata schema denoting/describing an entity, digital or otherwise		Phillipe		
```

Comment:
```suggestion
Metadata Record	A metadata record is an instance of a metadata schema denoting/describing an entity, digital or otherwise		Philippe		
```